src/data-tutorials/users.tsx

```diff
@@ -747,6 +747,13 @@ const Users: User[] = [
     website: 'blog/tutorials/como-construir-em-celo-usando-tatum',
     tags: ['beginner','celosage','ERC20','smartcontract','video'],
   },
+  {
+    title: 'Aprenda solidity e como criar os primeiros smart contract usando a IDE Remix',
+    description: 'Explicarei sobre conceitos de solidity e como criar seu primeiro smart contract.',
+    preview: require('./showcase/intermediate/Aprenda-solidity-e-como-criar-os-primeiros-smart-contract-usando-a-IDE-Remix.png)'),
```

Comment:
Please fix this typo, you have added a ")" at end of image path

Reply:
updated